sortinghat/parsing/grimoirelab.py

```diff
@@ -118,44 +119,49 @@ def __parse_identities(self, stream):
         """
         def __create_sh_identities(name, emails, yaml_entry):
             """Create SH identities based on name, emails and backens data in yaml_entry"""
-            my_ids = []
-            my_ids.append(Identity(name=name, source=self.source))
+
+            ids = []
+            ids.append(Identity(name=name, source=self.source))
 
             # FIXME we should encourage our users to add email or usernames
             # and if not returning at least a WARNING
             if emails:
                 for m in emails:
-                    my_ids.append(Identity(email=m, source=self.source))
+                    ids.append(Identity(email=m, source=self.source))
 
             for pb in PERCEVAL_BACKENDS:
-
                 if pb not in yaml_entry:
                     continue
 
                 for username in yaml_entry[pb]:
                     identity = Identity(username=username, source=pb)
-                    my_ids.append(identity)
-
-            return my_ids
+                    ids.append(identity)
 
+            return ids
 
         yaml = self.__load_yml(stream)
```

Comment:
Don't use 'yaml` name for variable. You are redefining this name. It was previously used when `yaml` was imported.